x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -216,7 +217,7 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
 
         try {
           const { listClient, exceptionsClient } = getListClient({
-            esClient: services.scopedClusterClient.asCurrentUser,
+            esClient,
```

Comment:
Should this also use the abortable client, or are you thinking that these queries should be "outside" the cancellable workflow?